UCLA Radio/UCLA Radio/menu/ShowViewController.swift

```diff
@@ -28,6 +28,29 @@ class ShowViewController: BaseViewController {
     @IBOutlet weak var scrollView: UIScrollView!
     @IBOutlet weak var columnContainerView: UIView!
 
+    @IBOutlet weak var facebookButton: UIButton!
+    @IBOutlet weak var tumblrButton: UIButton!
+    @IBOutlet weak var soundcloudButton: UIButton!
+    @IBOutlet weak var mixcloudButton: UIButton!
+
+    // Open the url when the button is pressed
+    /*@IBAction func facebookButtonHit(_ sender: AnyObject) {
+        UIApplication.shared.openURL(URL(string: facebook)!)
```

Comment:
I'd recommend one "buttonHit" function that uses the sender's id to determine what url to open. Less code to write!